axelrod/random_.py

```diff
@@ -3,7 +3,7 @@
 from axelrod import Actions
 
 
-def random_choice(p=0.5):
+def random_choice(p=0.5: float):
```

Comment:
Hello @souravsingh I am a member of the Axelrod team :smile:!
I believe you should drop `:float` cause `p` is determined. That should be the syntax error